libs/internal/include/launchdarkly/data_model/flag.hpp

```diff
@@ -83,5 +83,7 @@ struct Flag {
     bool trackEvents;
     bool trackEventsFallthrough;
     std::optional<std::uint64_t> debugEventsUntilDate;
+
+    [[nodiscard]] inline std::uint64_t Version() const { return version; }
```

Comment:
Confused.

Reply:
Sorry, this deserved documentation - it's satisfying the `ItemDescriptor` template requirements. Will add a comment.